pkg/tenant/db/time.go

```diff
@@ -0,0 +1,11 @@
+package db
+
+import "time"
+
+// Helper to convert a time.Time to an RFC3339Nano string for JSON serialization.
+func TimeToString(t time.Time) string {
+	if t.IsZero() {
+		return ""
+	}
+	return t.Format(time.RFC3339Nano)
```

Comment:
This is the one functional change. It shouldn't impact anything but it just makes sure that uninitialized `time.Time` values in the Created and Modified timestamps aren't accidentally formatted into strings for the frontend.

Reply:
Nice!